client/blocks/eligibility-warnings/hold-list.tsx

```diff
@@ -112,34 +124,43 @@ export const HoldList = ( { holds, isPlaceholder, siteSlug, translate } ) => {
 					</div>
 				) }
 				{ ! isPlaceholder &&
-					map( holds, hold => (
-						<div className="eligibility-warnings__hold" key={ hold }>
-							<Gridicon icon="notice-outline" size={ 24 } />
-							<div className="eligibility-warnings__message">
-								<span className="eligibility-warnings__message-title">
-									{ holdMessages[ hold ].title }
-								</span>
-								:&nbsp;
-								<span className="eligibility-warnings__message-description">
-									{ holdMessages[ hold ].description }
-								</span>
-							</div>
-							{ holdMessages[ hold ].supportUrl && (
-								<div className="eligibility-warnings__action">
-									<Button
-										compact
-										href={ holdMessages[ hold ].supportUrl }
-										rel="noopener noreferrer"
-									>
-										{ translate( 'Help' ) }
-									</Button>
+					map( holds, hold =>
+						! isKnownHoldType( hold, holdMessages ) ? null : (
```

Comment:
Now it makes me think we should log something when `hold` isn't a known hold type :-)

Reply:
Yeah with TS making us acknowledge where all the runtime errors are, it makes me think we should use something like sentry.io